dbt/adapters/bigquery/impl.py

```diff
@@ -21,11 +22,19 @@
 import google.cloud.exceptions
 import google.cloud.bigquery
 
+try:
+    # Library only needed for python models
+    from google.cloud import dataproc_v1
+    from google.cloud import storage
```

Comment:
- Let's add these as an `extras_require` — e.g. `pip install dbt-bigquery[dataproc]`
- Let's raise a clearer error below if a user wants to use Python models, and hasn't installed the extra